README.md

```diff
@@ -32,6 +32,8 @@
 
 ## TOC
 
+[![Greenkeeper badge](https://badges.greenkeeper.io/middyjs/middy.svg)](https://greenkeeper.io/)
```

Comment:
Can we move this badge closer to the other ones (the part in plain html)?

Reply:
fixed both things